person.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'nameable'
+class Person < Nameable
+  attr_accessor :id, :name, :age, :rental
+
+  def initialize(age, name = 'Unknown', _parent_permission = 1)
+    super()
+    @age = age
+    @name = name
+    @parent_permission = 1
+    @rental = []
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(book, date)
+    new_rental = Rental.new(date, self, book)
+    @rental << new_rental
+    book.rentals << new_rental
```

Comment:
- [ ] [OPTIONAL] Since we are adding the rental instance to the array inside its constructor, I don't think there is a need for the two lines `23` and `24`. I think instantiating the `Rental` class should be sufficient to complete the relationship. 🤝️

- [ ] [OPTIONAL] Please modify the `add_rental` method inside the book class to the same format. 🤓️

**Result**
```
def add_rental(book, date)
   Rental.new(date, self, book)
end
```